pages/advocates/index.vue

```diff
@@ -121,6 +123,10 @@ import CompactFeature from '~/components/features/CompactFeature.vue'
   }
 })
 export default class extends Vue {
+  title: string = 'Qiskit Advocates'
```

Comment:
In order to avoid repetition, we should be able to compute this field from the `head()` method.